app/res/layout/screen_login.xml

```diff
@@ -1,48 +1,56 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent" 
-    android:id="@+id/screen_login_main">
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:id="@+id/screen_login_main"
+                android:background="@color/cc_core_bg"
+    >
 
     <LinearLayout
-        android:id="@+id/screen_login_bottom_pane"
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:orientation="vertical" android:layout_alignParentBottom="true">
-
-        <TextView
-            android:id="@+id/str_version"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:textAppearance="@style/VersionIdText" />
-
-        <Button
-            android:id="@+id/login_button"
-            android:layout_width="fill_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="bottom"
-            android:nextFocusUp="@+id/edit_password" />
+    android:id="@+id/screen_login_bottom_pane"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical" android:layout_alignParentBottom="true">
+
+    <TextView
+    android:id="@+id/str_version"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:textAppearance="@style/VersionIdText"
+    android:visibility="gone" />
+
+    <!--<Button-->
+    <!--android:id="@+id/login_button"-->
+    <!--android:layout_width="fill_parent"-->
+    <!--android:layout_height="wrap_content"-->
+    <!--android:layout_gravity="bottom"-->
+    <!--android:nextFocusUp="@+id/edit_password" />-->
     </LinearLayout>
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_above="@id/screen_login_bottom_pane"
         android:layout_alignParentTop="true"
         android:orientation="vertical" >
 
         <FrameLayout
             android:layout_width="fill_parent"
-            android:layout_height="0px" android:layout_weight=".2" android:id="@+id/screen_login_banner_pane">
+            android:layout_height="0px"
+            android:layout_weight=".2"

```

Comment:
do we think this is always the right weight? might be useful to think about a max_height or min_height depending on the screen size.